Description: Fix for natty build 
 Avoid GCC 4.5 warning with 'case -1' statement on enum type.
Author: Kunal Goel <kunal.goel@linaro.org>
Last-Update: 2011-01-10
Index: libomxil-bellagio/src/base/omx_base_component.c
===================================================================
--- libomxil-bellagio.orig/src/base/omx_base_component.c	2011-06-02 11:27:40.758388019 +0800
+++ libomxil-bellagio/src/base/omx_base_component.c	2011-06-02 11:28:55.632985147 +0800
@@ -914,7 +914,7 @@
   if (ComponentParameterStructure == NULL) {
     return OMX_ErrorBadParameter;
   }
-  switch(nParamIndex) {
+  switch((int) nParamIndex) {
   case OMX_IndexParameterThreadsID:
 	    if ((err = checkHeader(ComponentParameterStructure, sizeof(OMX_PARAM_BELLAGIOTHREADS_ID))) != OMX_ErrorNone) {
 	      break;
